setup.py

```diff
@@ -294,6 +308,7 @@ def do_setup():
             'configparser>=3.5.0, <3.6.0',
             'croniter>=0.3.17, <0.4',
             'dill>=0.2.2, <0.3',
+            'dumb-init>=1.2.2',
```

Comment:
Likewise, this isn't needed at runtime if someone isn't installing it via docker, so this shouldn't be in this list.

Reply:
Ok. will try to find a better way (maybe use tini instead from apt-get).

Reply:
Fixed. Just moved it to devel